pkg/pillar/vault/handler_zfs.go

```diff
@@ -4,19 +4,26 @@
 package vault
 
 import (
+	"context"
 	"fmt"
+	"os"
+	"strings"
+	"time"
 
 	libzfs "github.com/bicomsystems/go-libzfs"
 	"github.com/lf-edge/eve-api/go/info"
 	"github.com/lf-edge/eve/pkg/pillar/base"
 	etpm "github.com/lf-edge/eve/pkg/pillar/evetpm"
 	"github.com/lf-edge/eve/pkg/pillar/types"
 	"github.com/lf-edge/eve/pkg/pillar/zfs"
+	"golang.org/x/sys/unix"
 )
 
 const (
-	zfsKeyDir  = "/run/TmpVaultDir2"
-	zfsKeyFile = zfsKeyDir + "/protector.key"
+	zfsKeyDir                       = "/run/TmpVaultDir2"
+	zfsKeyFile                      = zfsKeyDir + "/protector.key"
+	vaultZvolPathWaitSeconds int64  = 20
+	vaultFsType              string = "ext4"
```

Comment:
This name is potentially misleading. Does it make sense to name it zvolVaultFsType (or vaultZvolFsType)?

Also, do you have comments in the code and/or some markdown file (in pkg/pillar/docs) which explain that we use ext4 for the vault in the kubevirt flavor?